packages/core/src/domain/session/startSessionRenewTelemetry.ts

```diff
@@ -0,0 +1,35 @@
+import { getCookie } from '../../browser/cookie'
+import type { TimeStamp } from '../../tools/utils/timeUtils'
+import { elapsed, ONE_SECOND, timeStampNow } from '../../tools/utils/timeUtils'
+import { addTelemetryDebug } from '../telemetry'
+import type { SessionState } from './sessionState'
+import { SESSION_STORE_KEY } from './storeStrategies/sessionStoreStrategy'
+
+const FAST_RENEW_INTERVAL = ONE_SECOND
+const FAST_RENEW_COUNT_THRESHOLD = 2
+
+export function startSessionRenewTelemetry() {
+  let lastRenewTime = timeStampNow()
+  let fastRenewCount = 0
+
+  return {
+    onRenew(sessionCache: SessionState, sessionState: SessionState) {
+      const now = timeStampNow()
+      const interval = elapsed(lastRenewTime, now)
+      if (interval < FAST_RENEW_INTERVAL) {
+        fastRenewCount += 1
```

Comment:
Should we not reset the `fastRenewCount` if the renewal was correct?

```
|--|--------|--|
   ^           ^
 detected    ignored
```

Reply:
I think being notified once per page load is enough. Wdyt?